content/dart/concepts/switch/operators/operators.md

```diff
@@ -0,0 +1,128 @@
+---
+Title: 'operators.md #4242'
+
+Description: 'Operators in Dart are special symbols or phrases used to perform operations on variables and values.'
+
+Subjects: 'These operators are categorized into several types based on their functionality'
+- 'arithmetic'
+- 'equality and relational'
+- 'type test'
+- 'logical' 
+- 'bitwise'
+- 'assignment operators'
+
+Tags: 
```

Comment:
Tags should come from this [file](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).